deploy/webhook/cloudbuild.yaml

```diff
@@ -0,0 +1,31 @@
+steps:
+  # Get cluster credentials
+- name: gcr.io/k8s-skaffold/skaffold:v0.16.0
+  args: 
+  - 'gcloud'
+  - 'container'
+  - 'clusters'
+  - 'get-credentials'
+  - 'docs'
+  - '--zone'
+  - 'us-west2-a'
+  - '--project'
+  - 'k8s-skaffold'
+  # Build and push the image with the :commit_sha tag
+- name: gcr.io/k8s-skaffold/skaffold:v0.16.0
+  args:
+  - 'skaffold'
+  - '-f'
+  - 'deploy/webhook/skaffold.yaml'
+  - 'run'
+  - '--tag'
+  - '${COMMIT_SHA}'
+  # Build and deploy the image with the :latest tag
+- name: gcr.io/k8s-skaffold/skaffold:v0.16.0
+  args:
+  - 'skaffold'
+  - '-f'
+  - 'deploy/webhook/skaffold.yaml'
+  - 'run'
+  - '--tag'
+  - 'latest'
```

Comment:
Sorry I didn't follow the whole design, but what is :latest used for?

Reply:
I just realised that's left over from the previous design and not needed anymore! I'll remove it.